Gemfile.lock

```diff
@@ -249,25 +249,25 @@ GEM
     mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
     msgpack (1.3.3)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
-    net-ssh (6.0.2)
+    net-ssh (6.1.0)
     netrc (0.11.0)
     nio4r (2.5.4)
-    nokogiri (1.10.9)
+    nokogiri (1.10.10)
       mini_portile2 (~> 2.4.0)
-    oj (3.10.6)
+    oj (3.10.14)
     orm_adapter (0.5.0)
-    parallel (1.19.1)
+    parallel (1.19.2)
```

Comment:
### [parallel](https://github.com/grosser/parallel)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.19.1 → 1.19.2

[_~~change-log~~, [source-code](https://github.com/grosser/parallel/tree/v1.19.2)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/grosser/parallel/compare/v1.19.1...v1.19.2).

These are the individual commits:
- (82b2d5a) [docs: Improve notes about ActiveRecord](https://github.com/grosser/parallel/commit/82b2d5ae0797efe1a133799c94e35257bc069bcc)
- (d43d4d9) [Merge pull request #272 from kakra/fixes/issue-150](https://github.com/grosser/parallel/commit/d43d4d9a122fc1e5b3f15d6cf3d3d8a53b084b3b)
- (667fa9d) [simplify activerecord wording](https://github.com/grosser/parallel/commit/667fa9d777321ed6a57ace0f02c30d9fea7f0e1e)
- (df31eb8) [Allow timeout usage inside of threads](https://github.com/grosser/parallel/commit/df31eb8eb6ba23395aa59e583998b2f4f53f9b14)
- (fa8fa2b) [Merge pull request #277 from grosser/grosser/timeout](https://github.com/grosser/parallel/commit/fa8fa2bedba3cf6ccdee4beb73198c27e0827e82)
- (465f21d) [v1.19.2](https://github.com/grosser/parallel/commit/465f21d622943d52fea5bc512dbeb84828128d58)


</details>